examples/flower-simulation-step-by-step-pytorch/README.md

```diff
@@ -1,50 +1,127 @@
+In this tutorial series, we make reference to several pages in the [Flower Documentation](https://flower.ai/docs/). In particular, this videos highlight pages for:
 
-```bash
-# I'm assuming you are running this on an Ubuntu 22.04 machine (GPU is not required)
+- [Visualizing Dataset Distributions using `flwr-datasets`](https://flower.ai/docs/datasets/tutorial-visualize-label-distribution.html)
+- [List of all Partitioners available in `flwr-datasets`](https://flower.ai/docs/datasets/ref-api/flwr_datasets.partitioner.html)
+- [How-to Run Simulations page](https://flower.ai/docs/framework/how-to-run-simulations.html)
+- [How-to Design Stateful ClientApps](https://flower.ai/docs/framework/how-to-design-stateful-clients.html)
+- [Quickstart Pytorch Tutorial](https://flower.ai/docs/framework/tutorial-quickstart-pytorch.html)
+- [Advanced PyTorch Example](https://github.com/adap/flower/tree/main/examples/advanced-pytorch)
 
-# create the environment
-conda create -n flower_tutorial python=3.9 -y
+## Getting Started
 
-# activate your environment (depending on how you installed conda you might need to use `conda activate ...` instead)
-source activate flower_tutorial
+> \[!TIP\]
+> If you are developing on Windows, it is recommended to make use of the Windows Subsystem for Linux (WSL). Check the guide on [how to setup WSL for development on Windows](https://code.visualstudio.com/docs/remote/wsl).
 
-# install PyToch (other versions would likely work)
-conda install pytorch==1.13.1 torchvision==0.14.1 torchaudio==0.13.1 pytorch-cuda=11.6 -c pytorch -c nvidia -y
-# conda install pytorch==1.13.1 torchvision==0.14.1 torchaudio==0.13.1 -c pytorch -y # If you don't have a GPU
+As presented in the video, we start from a new Python 3.11 environment. You only need to activate it and install `flwr`.
 
-# Install Flower and other dependencies
-pip install -r requirements.txt
+```bash
+# Install Flower
+pip install -U flwr
 ```
 
-If you are running this on macOS with Apple Silicon (i.e. M1, M2), you'll need a different `grpcio` package if you see an error when running the code. To fix this do:
+Then, use the `flwr new` command to construct a new Flower App using the PyTorch template. The vast majority of the content added to the App (as described in the video tutorials) isn't specific to PyTorch. This means you are welcome to choose another template such as `TF`, `NumPy`, `MLX` or `JAX` if you prefer. Just keep in mind you might need to do additional edits to your App (e.g. save the model differently, in the recommended way by your chosen ML framework) If you are undecided, `PyTorch` is a great framework:
 
-```bash
-# with your conda environment activated
-pip uninstall grpcio
+```shell
+flwr new my-awesome-app # then follow the prompt and choose PyTorch
```

Comment:
Suggestion: Should we move the code directly behind `Then, use the `flwr new` command to construct a new Flower App using the PyTorch template:`? This way it's more obvious how straight forward the installation is.